doc/source/cookbook/simple_projection_methods.py

```diff
@@ -0,0 +1,13 @@
+import yt
+
+# Load the dataset.
+ds = yt.load("GalaxyClusterMerger/fiducial_1to3_b0.273d_hdf5_plt_cnt_0175")
```

Comment:
So the script is currently broken (we can see that it breaks the docs build), so I suggest you iterate a bit more locally and check that it's working as you'd expect before pushing again. Additionally, this specific dataset is huge, running the recipe on my machine takes about a minute (and one of the three methods doesn't even work). I would suggest using a smaller dataset for this demo.
```suggestion
ds = yt.load("IsolatedGalaxy/galaxy0030/galaxy0030")
```

Reply:
Yup, this is a problem. Working on it now.

Reply:
got the solution, see my latest comment !